joomla/joomla-cms
# Fix for Codemirror Cant Type Forward Slash

This PR applies an upstream bug fix to CodeMirror
see: https://github.com/codemirror/CodeMirror/commit/076afcb14e8be6
# Steps to reproduce the issue

Set Codemirror as default editor.
Either create a new article or edit an existing article.
Type a forward slash ( / ) in the editor window.
# Expected result

A forward slash appears in the editor window
# What happens

No slash....
# To test this PR

Apply patch - set CodeMirror as active editor see if you can type a slash in editor window

Closes #6277
Closes #6181
